packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -427,11 +432,30 @@ export const useOrganizationProfileContext = (): OrganizationProfileContextType
   const navigateAfterLeaveOrganization = () =>
     navigate(ctx.afterLeaveOrganizationUrl || displayConfig.afterLeaveOrganizationUrl);
 
+  const isMembersPageRoot = pages.routes[0].id === ORGANIZATION_PROFILE_NAVBAR_ROUTE_ID.MEMBERS;
+  const isSettingsPageRoot = pages.routes[0].id === ORGANIZATION_PROFILE_NAVBAR_ROUTE_ID.SETTINGS;
+
+  const pathToDomainPage = useMemo(
+    () =>
+      isSettingsPageRoot
+        ? '../domain'
+        : isMembersPageRoot
+        ? './organization-settings/domain'
+        : '../organization-settings/domain',
+    [isMembersPageRoot, isMembersPageRoot],
+  );
```

Comment:
Why are we using `memo` for a simple ternary?

Reply:
i dropped it, as we were not gaining from having it